src/containers/Results/Results.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import './Results.css'
+// import * as actions from '../../actions';
+import { connect } from 'react-redux'
+import { Link } from "react-router-dom";
+
+const Results = ({solarData}) => {
+    const [savingsValue] = useState(50)
+    const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']
+
+    const tableData = months.map((item, index)=>{
+        return (
+            <tr key={item}>
+                <td>{item}</td>
+                <td>{Number(solarData.solrad_monthly[index]).toFixed(2)}</td>
+                <td>{Number(solarData.ac_monthly[index]).toFixed(2)}</td>
+                <td>{Number(solarData.ac_monthly[index] * .11).toFixed(2)}</td>
```

Comment:
Note that we anticipate this calculation to eventually be performed on the BE, and will replace the contents of this td with `Number(solarData.value_monthly[index].toFixed(2)`